osc/commandline.py

```diff
@@ -2324,9 +2324,9 @@ def do_request(self, subcmd, opts, *args):
 
         apiurl = self.get_api_url()
 
-        if cmd in ['list']:
+        if cmd in ('list'):
```

Comment:
This is incorrect, the parentheses are ignored and the following is evaluated: `if cmd in 'list'`
To use a tuple properly, you need to add a trailing comma: `if cmd in ('list', )` or compare the value directly: `if cmd == 'list'`

Reply:
sure, fixed